packages/flame_bloc/test/flame_bloc_game_test.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use_from_same_package
+
```

Comment:
Why is that?

Reply:
Because I didn't removed the old API, just deprecated them and here are their tests to keep coverage.